includes/class-sensei-course.php

```diff
@@ -132,6 +132,9 @@ public function __construct() {
 		// ensure the course category page respects the manual order set for courses
 		add_filter( 'pre_get_posts', array( __CLASS__, 'alter_course_category_order' ), 10, 1 );
 
+		// filter the redirect url after enrolment
```

Comment:
```suggestion
		// Filter the redirect url after enrolment.
```

A fix for the PHPCS issue. All the comments should end with a period. ;)